package.json

```diff
@@ -4,10 +4,12 @@
   "private": true,
   "dependencies": {
     "bulma": "^0.8.0",
+    "lint-fix": "^0.24.0",
```

Comment:
There is no place we use lint-fix please remove it

Reply:
lint-fix removed